content/en/docs/Getting Started/pipelines.md

```diff
@@ -14,8 +14,7 @@ This tutorial shows you how to:
 +   Create a Pipeline containing your Tasks.
 +   Use `PipelineRun` to instantiate and run the Pipeline containing your Tasks.
 
-For this tutorial we are going to use [minikube][minikube] to run the commands
-locally.
+This guide uses a local cluster with [minikube][].
```

Comment:
is the empty [] intentional?

Reply:
Yep, in reference-style links if you omit the reference it picks up what's in the first bracket instead.

Reply:
TIL!